cms/src/main/java/uk/nhs/digital/apispecs/swagger/CodegenDefaultGenerator.java

```diff
@@ -0,0 +1,51 @@
+package uk.nhs.digital.apispecs.swagger;
+
+import static freemarker.template.utility.Collections12.singletonList;
```

Comment:
I've only just noticed: this wants to be `import static java.util.Collections.singletonList;`